plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -57,41 +67,66 @@
     required: false
     type: bool
   merge:
     description:
-     - Will run out-of-order or missing migrations as they are not rollback migrations, you can only use this parameter with 'migrate' command
+     - Will run out-of-order or missing migrations as they are not rollback migrations, you can only use this
+       parameter with C(migrate) command
     required: false
     type: bool
   link:
     description:
-     - Will create links to the files instead of copying them, you can only use this parameter with 'collectstatic' command
+     - Will create links to the files instead of copying them, you can only use this parameter with
+       C(collectstatic) command
     required: false
     type: bool
+  liveserver:
+    description:
+      - This parameter was implemented a long time ago in a galaxy far way. It probably relates to the
+        django-liveserver package, which is no longer updated.
+      - Hence, it will be considered DEPRECATED and should be removed in a future release.
+    type: str
+    required: false
+    aliases: [live_server]
+  testrunner:
+    description:
+      - "From the Django docs: Controls the test runner class that is used to execute tests"
+      - This parameter is passed as-is to C(manage.py)
+    type: str
+    required: false
+    aliases: [test_runner]
 notes:
-  - I(virtualenv) (U(http://www.virtualenv.org)) must be installed on the remote host if the virtualenv parameter is specified.
-  - This module will create a virtualenv if the virtualenv parameter is specified and a virtualenv does not already exist at the given location.
-  - This module assumes English error messages for the 'createcachetable' command to detect table existence, unfortunately.
-  - To be able to use the migrate command with django versions < 1.7, you must have south installed and added as an app in your settings.
-  - To be able to use the collectstatic command, you must have enabled staticfiles in your settings.
-  - As of ansible 2.x, your I(manage.py) application must be executable (rwxr-xr-x), and must have a valid I(shebang), i.e. "#!/usr/bin/env python",
-    for invoking the appropriate Python interpreter.
+  - C(virtualenv) (U(http://www.virtualenv.org)) must be installed on the remote host if the virtualenv parameter
+    is specified.
+  - This module will create a virtualenv if the virtualenv parameter is specified and a virtualenv does not already
+    exist at the given location.
+  - This module assumes English error messages for the C(createcachetable) command to detect table existence,
+    unfortunately.
+  - To be able to use the C(migrate) command with django versions < 1.7, you must have C(south) installed and added
+    as an app in your settings.
+  - To be able to use the C(collectstatic) command, you must have enabled staticfiles in your settings.
+  - As of ansible 2.x, your C(manage.py) application must be executable (rwxr-xr-x), and must have a valid I(shebang),
```

Comment:
```suggestion
  - Your C(manage.py) application must be executable (rwxr-xr-x), and must have a valid C(shebang),
```
2.x is not very precise, and in any case seems to be a long time ago, so I guess we can just remove it :)

Reply:
C(shebang) => maybe B(shebang)? It is not "files and option values" as pointed in the docs.

Reply:
Sounds good!

Reply:
Come to think of it, I don't believe we need a markup on that term at all, especially because there is an example right after it.